lib/income_and_assets/submissions/monitor.rb

```diff
@@ -7,7 +7,7 @@ module IncomeAndAssets
   module Submissions
     class Monitor
       # statsd key for sidekiq
-      SUBMISSION_STATS_KEY = 'worker.income_and_assets_intake_job'
+      SUBMISSION_STATS_KEY = 'worker.lighthouse.income_and_assets_intake_job'
```

Comment:
does datadog reporting need to be updated?

Reply:
I don't think so. There shouldn't be any monitors or reports pulling I&A metrics just yet. Also, nothing in production is invoking these monitor functions just yet.